buildpacks/ruby/src/layers/bundle_install_layer.rs

```diff
@@ -184,58 +231,97 @@ impl Layer for BundleInstallLayer<'_> {
 
         match cache_state(old.clone(), now) {
             Changed::Nothing => {
-                log_step("Loading cache");
+                log_step("Loading cached gems");
 
                 keep_and_run
             }
-            Changed::Stack(_old, _now) => {
-                log_step(format!("Clearing cache {}", fmt::details("stack changed")));
+            Changed::DistroName(old, now) => {
+                log_step(format!(
+                    "Clearing cache {}",
+                    fmt::details(format!("distro name changed: {old} to {now}"))
+                ));
+
+                clear_and_run
+            }
+            Changed::DistroVersion(old, now) => {
+                log_step(format!(
+                    "Clearing cache {}",
+                    fmt::details(format!("distro version: {old} to {now}"))
```

Comment:
It seems like the "changed" word is missing from some of these but not all? ie: "distro name changed" vs "distro version". Whereas the strings in `buildpacks/ruby/src/layers/ruby_install_layer.rs` consistently include the word "changed".

Reply:
Updated. Thanks.